plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/FreshNames.kt

```diff
@@ -30,3 +30,8 @@ data object InvokeFunctionObjectName : MangledName {
     override val mangled: String
         get() = "special\$invoke_function_object"
 }
+
+data class InlineName(val inlineFunctionName: String, val name: String) : MangledName{
```

Comment:
Would it make sense to use `MangledName` for `name` (or even both), so you can mangle different ones?